### ISSUE TITLE
The NVIDIA Driver was not detected. 

### ISSUE THREAD

**Issue by blackcement** [NVlabs/mask-auto-labeler#15]
2023-07-04 23:10:20 
2023-07-04 23:10:20 =============
2023-07-04 23:10:20 == PyTorch ==
2023-07-04 23:10:20 =============
2023-07-04 23:10:20 
2023-07-04 23:10:20 NVIDIA Release 21.12 (build 29870972)
2023-07-04 23:10:20 PyTorch Version 1.11.0a0+b6df043
2023-07-04 23:10:20 
2023-07-04 23:10:20 Container image Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
2023-07-04 23:10:20 
2023-07-04 23:10:20 Copyright (c) 2014-2021 Facebook Inc.
2023-07-04 23:10:20 Copyright (c) 2011-2014 Idiap Research Institute (Ronan Collobert)
2023-07-04 23:10:20 Copyright (c) 2012-2014 Deepmind Technologies    (Koray Kavukcuoglu)
2023-07-04 23:10:20 Copyright (c) 2011-2012 NEC Laboratories America (Koray Kavukcuoglu)
2023-07-04 23:10:20 Copyright (c) 2011-2013 NYU                      (Clement Farabet)
2023-07-04 23:10:20 Copyright (c) 2006-2010 NEC Laboratories America (Ronan Collobert, Leon Bottou, Iain Melvin, Jason Weston)
2023-07-04 23:10:20 Copyright (c) 2006      Idiap Research Institute (Samy Bengio)
2023-07-04 23:10:20 Copyright (c) 2001-2004 Idiap Research Institute (Ronan Collobert, Samy Bengio, Johnny Mariethoz)
2023-07-04 23:10:20 Copyright (c) 2015      Google Inc.
2023-07-04 23:10:20 Copyright (c) 2015      Yangqing Jia
2023-07-04 23:10:20 Copyright (c) 2013-2016 The Caffe contributors
2023-07-04 23:10:20 All rights reserved.
2023-07-04 23:10:20 
2023-07-04 23:10:20 NVIDIA Deep Learning Profiler (dlprof) Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
2023-07-04 23:10:20 
2023-07-04 23:10:20 Various files include modifications (c) NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
2023-07-04 23:10:20 
2023-07-04 23:10:20 This container image and its contents are governed by the NVIDIA Deep Learning Container License.
2023-07-04 23:10:20 By pulling and using the container, you accept the terms and conditions of this license:
2023-07-04 23:10:20 https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license
2023-07-04 23:10:20 
2023-07-04 23:10:20 WARNING: The NVIDIA Driver was not detected.  GPU functionality will not be available.
2023-07-04 23:10:20    Use the NVIDIA Container Toolkit to start this container with GPU support; see
2023-07-04 23:10:20    https://docs.nvidia.com/datacenter/cloud-native/ .
2023-07-04 23:10:20 
2023-07-04 23:10:20 NOTE: MOFED driver for multi-node communication was not detected.
2023-07-04 23:10:20       Multi-node communication performance may be reduced.
2023-07-04 23:10:20 
2023-07-04 23:10:20 NOTE: The SHMEM allocation limit is set to the default of 64MB.  This may be
2023-07-04 23:10:20    insufficient for PyTorch.  NVIDIA recommends the use of the following flags:
2023-07-04 23:10:20    docker run --gpus all --ipc=host --ulimit memlock=-1 --ulimit stack=67108864 ...

The driver is installed, CUDA too, neural networks work fine in python, but this package does not see the video card, why is that?
Can you suggest what is the problem?

**Comment by voidrank** [comment]
Hi @blackcement  please use nvidia-docker

**Comment by blackcement** [comment]
> Hi @blackcement please use nvidia-docker

@voidrank , Thanks, but which image should I take? There, at the request of their whole mountain falls out?
![image](https://github.com/NVlabs/mask-auto-labeler/assets/61663301/d43fbbfd-6603-417e-9947-e0a29664da54)

**Comment by voidrank** [comment]
https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html
